userspace/libsinsp/filterchecks.cpp

```diff
@@ -5943,6 +5944,26 @@ uint8_t* sinsp_filter_check_event::extract(sinsp_evt *evt, OUT uint32_t* len, bo
 			RETURN_EXTRACT_VAR(m_u32val);
 		}
 		break;
+	case TYPE_ISOPEN_CREATE:
```

Comment:
Would it be possible to fall through in the existing cases to have less code duplication? Then we would also inherit the existing evt type checks that are currently missing.

If you see opportunities to optimize the old code, please do :), e.g. early returns or only do stuff when relevant.

```
case TYPE_ISOPEN_READ:
case TYPE_ISOPEN_WRITE:
case TYPE_ISOPEN_EXEC:
case TYPE_ISOPEN_CREATE:
```